### PR TITLE
mon/MonCommands: add "features" in "report"

### PR DESCRIPTION
**Pull request by dcasier**
Signed-off-by: David Casier <david.casier@aevoo.fr>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`

</details>

### CODE REVIEW

**Comment by dcasier** [comment]
it is possible to restart the make in error or I have to push "dummy" ?

File: qa/workunits/cephtool/test.sh [hunk 1]
```diff
@@ -1179,6 +1179,10 @@ function test_mon_mon()
   ceph mon feature set kraken --yes-i-really-mean-it
   expect_false ceph mon feature set abcd
   expect_false ceph mon feature set abcd --yes-i-really-mean-it
+
+  report=$(ceph report -f json | jq -c '.features')
+  features=$(ceph features -f json | jq -c '.')
+  [[ "$report" == "$features" ]]
```
Comment (tchaikov) [inline qa/workunits/cephtool/test.sh]:
```
2019-10-26T00:31:06.665 INFO:tasks.workunit.client.0.mira032.stderr://home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1183: test_mon_mon:  ceph report -f json
2019-10-26T00:31:06.665 INFO:tasks.workunit.client.0.mira032.stderr://home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1183: test_mon_mon:  jq -c .features
2019-10-26T00:31:07.454 INFO:tasks.workunit.client.0.mira032.stderr:report 2946713994
2019-10-26T00:31:07.480 INFO:tasks.workunit.client.0.mira032.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1183: test_mon_mon:  report='{"mon":[{"features":"0x3f01cfb8ffadffff"
,"release":"luminous","num":1}],"client":[{"features":"0x3f01cfb8ffadffff","release":"luminous","num":1}]}'
2019-10-26T00:31:07.481 INFO:tasks.workunit.client.0.mira032.stderr://home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1184: test_mon_mon:  ceph features -f json
2019-10-26T00:31:07.482 INFO:tasks.workunit.client.0.mira032.stderr://home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1184: test_mon_mon:  jq -c .
2019-10-26T00:31:08.286 INFO:tasks.workunit.client.0.mira032.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1184: test_mon_mon:  features='{"mon":[{"features":"0x3f01cfb8ffadfff
f","release":"luminous","num":3}],"osd":[{"features":"0x3f01cfb8ffadffff","release":"luminous","num":3}],"client":[{"features":"0x3f01cfb8ffadffff","release":"luminous","num":2}],"mgr":[{"features":"0x3f0
1cfb8ffadffff","release":"luminous","num":1}]}'
2019-10-26T00:31:08.287 INFO:tasks.workunit.client.0.mira032.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1185: test_mon_mon:  [[ {"mon":[{"features":"0x3f01cfb8ffadffff","rel
ease":"luminous","num":1}],"client":[{"features":"0x3f01cfb8ffadffff","release":"luminous","num":1}]} == \{\"\m\o\n\"\:\[\{\"\f\e\a\t\u\r\e\s\"\:\"\0\x\3\f\0\1\c\f\b\8\f\f\a\d\f\f\f\f\"\,\"\r\e\l\e\a\s\e\
"\:\"\l\u\m\i\n\o\u\s\"\,\"\n\u\m\"\:\3\}\]\,\"\o\s\d\"\:\[\{\"\f\e\a\t\u\r\e\s\"\:\"\0\x\3\f\0\1\c\f\b\8\f\f\a\d\f\f\f\f\"\,\"\r\e\l\e\a\s\e\"\:\"\l\u\m\i\n\o\u\s\"\,\"\n\u\m\"\:\3\}\]\,\"\c\l\i\e\n\t\"\
:\[\{\"\f\e\a\t\u\r\e\s\"\:\"\0\x\3\f\0\1\c\f\b\8\f\f\a\d\f\f\f\f\"\,\"\r\e\l\e\a\s\e\"\:\"\l\u\m\i\n\o\u\s\"\,\"\n\u\m\"\:\2\}\]\,\"\m\g\r\"\:\[\{\"\f\e\a\t\u\r\e\s\"\:\"\0\x\3\f\0\1\c\f\b\8\f\f\a\d\f\f\
f\f\"\,\"\r\e\l\e\a\s\e\"\:\"\l\u\m\i\n\o\u\s\"\,\"\n\u\m\"\:\1\}\]\} ]]
2019-10-26T00:31:08.287 INFO:tasks.workunit.client.0.mira032.stderr:/home/ubuntu/cephtest/clone.client.0/qa/workunits/cephtool/test.sh:1: test_mon_mon:  rm -fr /tmp/cephtool.UA0
2019-10-26T00:31:08.289 DEBUG:teuthology.orchestra.run:got remote process result: 1
```

**Comment by tchaikov** [comment]
if "features" is already able to report "features", why do you want to report it in "report"?

**Comment by dcasier** [comment]
> if "features" is already able to report "features", why do you want to report it in "report"?

In order to retrieve a maximum of information on the state of the cluster, in the case of a request for support, via the "ceph report" command. But nothing blocking if the pull request does not seem interesting.

**Pull request closed without merging**